types/index.d.ts

```diff
@@ -80,7 +79,7 @@ export class Canvas {
 
 	constructor(width: number, height: number, type?: 'image'|'pdf'|'svg')
 
-	getContext(contextId: '2d', contextAttributes?: NodeCanvasRenderingContext2DSettings): NodeCanvasRenderingContext2D
+	getContext(contextId: '2d', contextAttributes?: NodeCanvasRenderingContext2DSettings): CanvasRenderingContext2D
```

Comment:
This probably means a semver-major release, right?

Reply:
Hmm, might be worth keeping the old name then? 🤔

Reply:
We could also export aliases to the old names. I thought the "NodeCanvas" prefix was to avoid naming conflicts from `lib.dom.ts`.

Even then, it still could break someone's build if they were making function calls with arguments we ignore. Not sure if that's as big of a deal though?

Reply:
I'm not at all opposed to a semver-major release. This would be an easy one for users to handle. Just want to be on the same page.

Other question is usability - users could alias the type themselves (`import type {CanvasRenderingContext2D as NodeCanvasRenderingContext2D}`), but is it more convenient for the provided name to be NodeCanvasRenderingContext2D?

Reply:
I prefer the shorter name. We call it Canvas everywhere else, not NodeCanvas. Our main users are using node, so it shouldn't cause a naming conflict should it? If they do have DOM types, the imported type would take priority over the ambient types, which is what you'd expect. I'm not 100% certain it's the right call, but pretty sure it would be fine...

Reply:
Plus we already kind did this. You can `import {Image}` which would conflict with the browser-defined `Image`.

Reply:
Fair point, same for the the gradient and pattern classes.

@LinusU  good with you? Shorter name and semver-major?

Reply:
Hmm, it seems like the old code exported without the `Node` prefix, so I'm not sure this is a breaking change?

https://github.com/Automattic/node-canvas/blob/672104c1a4bd202e56d8837ef83ebf7aee2dfce2/types/index.d.ts#L210

Reply:
That's true. I think it would still let you `import {NodeCanvasRenderingContext2D} from 'canvas'` because it's a .d.ts. Could still export both. IMO the chances of this breaking someone's build are small as-is and we can only make it smaller but not get to 0.

Reply:
Good point! Makes more sense as semver-minor then.